theme/static/style/style.css

```diff
@@ -454,6 +454,75 @@ pre code {
     }
 }
 
+@media screen and (max-width: 991px){
+    /* For tablets: */
+    /*Screenshots section.*/
+    .arrow-left, .arrow-right, .img-index {
+        display: none;
+        width: 0;
+        height: 0;
+    }
+
+    .img-wrapper {
+        width: 100%;
+    }
+
+    .img {
+        bottom: 3rem;
+    }
+}
+
+@media screen and (max-width: 767px) {
+    /* For mobile phones: */
+    /*screenshots section*/
+    .arrow-left, .arrow-right, .img-index {
+        display: none;
+        width: 0;
+        height: 0;
+    }
+
+    .img-wrapper {
+        width: 100%;
+    }
+
+    .screenshot-title {
+        padding: 1% 0;
```

Comment:
Use padding 1.8rem same as in other sections. Don't make up measurements on your own.